frontend/tests/jest/__mocks__/products.ts

```diff
@@ -690,7 +678,29 @@ export const mockedProduct: Product = {
          enabled: true,
       },
    ],
-   sections: getDefaultProductSections(),
+   sections: [
+      {
+         type: 'ProductOverview',
+         id: 'ProductOverviewSection-0',
+      },
+      {
+         type: 'ReplacementGuides',
+         id: 'ReplacementGuidesSection-1',
+         title: null,
+         guides: [
+            {
+               id: '0',
+               title: 'iPhone 6s Plus Battery Replacement',
+               url: '/Guide/iPhone+6s+Plus+Battery+Replacement/51380',
+               imageUrl:
+                  'https://mmcelvain.cominor.com/igi/LVQpdSdCEY1YxPkM.thumbnail',
```

Comment:
Ideally we should set the images to www.cominor.com rather than a specific developer like mmcelvain but it probably doesn't really matter.